source/ui/beamline/AMDeadTimeButton.cpp

```diff
@@ -142,3 +187,48 @@ bool AMDeadTimeButton::hasICRDataSource() const
 {
 	return !(inputCountSource_ == 0);
 }
+
+bool AMDeadTimeButton::hasAcquireTime() const
+{
+	return !(acquireTimeControl_ == 0);
+}
+
+double AMDeadTimeButton::getPercent() const
+{
+	double result = 0;
+
+	if (canDisplayPercentage())
+		result = 100*(1 - double(outputCountSource_->value(AMnDIndex()))/double(inputCountSource_->value(AMnDIndex())));
```

Comment:
Same thing here. `canDisplayPercentage()` checks whether the outputCountSource_ and inputCountSource_ are 0. I'll modify to make this explicit/more clear.